artiq/frontend/artiq_influxdb.py

```diff
@@ -11,7 +11,10 @@
 import numpy as np
 import aiohttp
 
-from artiq.tools import *
+from artiq.tools import (
```

Comment:
In general and in this case specifically I prefer to avoid star imports. It adds clarity here and for the general case there are plenty of reasons around.

Reply:
`import *` doesn't meet `flake8` as laid out in contribution guidelines

Reply:
Then the contribution guidelines should be changed.

Reply:
Please don't. Just saying flake8 resolves so many mostly subjective discussions. I haven't seen an objection to the flake8 style that merits the time spent arguing and deciding it by consensus in each case.

Reply:
Here is one: ``import *`` is more useful than following ``flake8`` precisely.